### ISSUE TITLE
Linking a related entity on object passed to $create() method try to create this entity

### ISSUE THREAD

**Issue by mrblackus** [Wakanda/wakanda-issues#14]
Using `$create()` directly passing an array of parameter, if I pass a related entity (to create reference to it) in this object, it will try to also create this related entity, rather than just referencing it, resulting on a 500 error from the server.

Example :

```javascript
author; //author is a valid Angular-Wakanda entity
var message = ds.Message.$create({
    text: $scope.newMessage,
    date: new Date(),
    author: author
});

message.$save(); //500 error, because trying to create an new author with the same ID and attributes
```

Just linking the entity outside of the object passed to `$create()` works normally.

```javascript
author; //author is a valid Angular-Wakanda entity
var message = ds.Message.$create({
    text: $scope.newMessage,
    date: new Date()
});

message.author = author;
message.$save(); //Works like a charm
```

Not sure if it's on purpose or a bug.

**Comment by yboukhata** [comment]
Hi,

With `message.author = author;`, the **related attribute author** of message Entity is setted (this setter is defined with Object.definePropery) explicitly to the author entity.

With the other creation, we need to made a fix.

**Comment by hamzahik** [comment]
Fixed in https://github.com/Wakanda/angular-wakanda/releases/tag/v1.0.2
Will be included in the next Wakanda release 1.0.1
